docs/guides/preinstall.md

```diff
@@ -4,77 +4,111 @@ These steps are common to most distros. Please refer to both this page and this
 
 !!! Warning "A software update is required to use this startup disk"
-    If you see this message when trying to boot Linux, Apple may be enforcing secure boot. Make sure you have it [turned off](#disable-secure-boot), and if there were two Orange "EFI Boot" entries in Startup Manager, try selecting the other one. This occurs when booting Linux from a partition formatted as APFS or "macOS Extended" (also known as "hfs+") even when Secure Boot is completely disabled.
+    If you see this message when trying to boot Linux, Apple may be enforcing Secure Boot. Make sure you have [disabled Secure Boot](#disable-secure-boot). If there were two orange "EFI Boot" entries in Startup Manager, try selecting the other one. This may occur when booting Linux from a partition formatted as APFS or "macOS Extended" (also known as "hfs+"), even when Secure Boot is completely disabled.
 
 ## Follow distro specific steps
 
-Once you've selected your USB, it should boot your distro's install image. Now follow the installation guide of your specific distribution.
+You can now follow the installation guide for your chosen distribution: [Guides for different distributions](https://wiki.t2linux.org/distributions/overview/)
 
 !!! Warning "Automatic Partitioning"
-    The installers of many distros provide options like "Automatic Partitioning" when installing Linux. You always have to select the option that allows "Manual Partitioning". **Do not** select "Automatic Partitioning" as it shall remove macOS as well. During manual partitioning make sure that you mount `/dev/nvme0n1p1` or your [separate EFI partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions) (whatever applies to your case) at `/boot/efi` and the partition you created for Linux at `/`. If you want separate partitions for `/home`, `/boot` etc., make sure you have created them before as well and mount them accordingly during manual partitioning.
-
-This wiki provides a set of [guides for different distributions](https://wiki.t2linux.org/distributions/overview/). If the distribution you want to use is present there, it's recommended to follow it instead of the official documentation by distribution vendor, as it considers T2 support.
+    The installers of many distros will provide options like "Automatic Partitioning" when installing Linux. **Do not use** "Automatic Partitioning" as it may remove macOS as well. You **always** must select the option that allows **"Manual Partitioning"**. Make sure that: **1.** you mount your EFI `/dev/nvme0n1p1`  at `/boot/efi` (or if you use a [separate EFI partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions), at whatever applies to your case). **2.** you mount your partition for Linux at `/`. **3.** if you created separate partitions for `/home`, `/boot`, etc.. you mount each of them correctly according to their mount points.
```

Comment:
This change can be reverted

Reply:
You reverted the wrong commit. The change for debian was fine. You need to revert the change I've pointed out in this review.

Reply:
reverted [1418a99](https://github.com/gianmaticus/wiki/commit/1418a99717e6f2a726d34932a8c4a6e8eaafbbe9)